module-5/src/services/usersService.js

```diff
@@ -0,0 +1,102 @@
+import { randomUUID } from "crypto";
+import fs from "fs";
+
+const readFromDatabase = () => {
+  try {
+    return JSON.parse(
+      fs.readFileSync(new URL("../database/db.json", import.meta.url))
+    );
+  } catch {
+    throw {
+      status: 503,
+      message: `Error on reading data from the database`,
+    };
+  }
+};
+
+const writeToDatabase = (DB) => {
+  try {
+    fs.writeFileSync(
```

Comment:
Switch to fs.writeFile for non-blocking I/O, enhancing performance

The synchronous file operations (`readFileSync`, `writeFileSync`) can block the Node.js event loop, affecting performance. Consider using their asynchronous counterparts (`readFile`, `writeFile`) along with `async/await` for non-blocking I/O operations